ditto/cli.py

```diff
@@ -98,32 +109,28 @@ def metric(ctx, **kwargs):
             "Cannot read from format '{}'".format(kwargs["from"])
         )
 
-    if kwargs["input"] is None:
-        raise click.BadOptionUsage(
-            "input",
-            "--input must be provided."
-        )
-
-    from_reader_name = kwargs["from"]
-
-    try:
-        MetricComputer(
-            registered_reader_class=_load(registered_readers, from_reader_name),
-            input_path=kwargs["input"],
-            output_format=kwargs["to"],
-            output_path=kwargs["output"],
-            by_feeder=kwargs["feeder"],
-        ).compute()
-    except Exception as e:
-        # TODO: discuss whether we should raise exception here?
```

Comment:
This behavior was non-ideal.  Any exception would cause the application to exit with no error message.  The change lets it fail naturally.  As it happens, there is a bug in the code that was causing this confusing behavior.